src/natcap/invest/managed_raster/ManagedRaster.h

```diff
@@ -337,15 +340,15 @@ class ManagedRaster {
         }
 };
 
+template<class T>
```

Comment:
This is a class template. `ManagedFlowDirRaster` is _not_ a class, but `ManagedFlowDirRasater<MFD>` and `ManagedFlowDirRasater<D8>` are classes that can be created from this template.